timely_beliefs/beliefs/utils.py

```diff
@@ -620,7 +620,7 @@ def read_csv(
         df = pd.read_excel(path, **kwargs)  # requires openpyxl
     else:
         raise TypeError(
-            f"Extension {ext} not recognized. Accepted file extensions are csv, xlsx and xls."
+            f"Extension {ext} not recognized. Accepted file extensions are csv, xlsm, xlsx and xls."
```

Comment:
If you could also add "xlsm" to the list in line 619, then it would also be picked up as an Excel file when attempting to read it.

Reply:
done @Flix6x